public-ui/src/app/data-browser/charts/chart-biosex/chart-biosex-react.component.tsx

```diff
@@ -0,0 +1,257 @@
+        }
+        data = data.sort((a, b) => {
+          if (a.name > b.name) {
+            return 1;
+          }
+          if (a.name < b.name) {
+            return -1;
+          }
+          return 0;
+        }
+        );
+        cats = cats.sort((a, b) => {
+          if (a > b) {
+            return 1;
+          }
+          if (a < b) {
+            return -1;
+          }
+          return 0;
+        });
+        const temp = data.filter(x => x.y > 20);
+        const dataOnlyLT20 = temp.length > 0 ? false : true;
+        const series = [
+          {
+            color: '#2691D0',
+            legendColor: '#2691D0',
+            name: legendText, colorByPoint: false, data: data, dataOnlyLT20: dataOnlyLT20
+          }];
+        return { categories: cats, series: series};
+  }
+
+  prepFitbitCategoriesAndData() {
+    let pointData = [];
+    let categoryArr = [];
+    for (const concept of this.props.genderAnalysis.results) {
+          const genderCountResults = this.props.genderCountAnalysis.results.filter(r =>
+          r.stratum4 === concept.stratum2);
+          let genderCountTooltip = '';
+          let percentage;
+          if (genderCountResults && genderCountResults.length > 0) {
+                percentage = ((concept.countValue /
+                genderCountResults[0].countValue) * 100).toFixed();
+                if (percentage < 1) {
+                    percentage =
+                    ((concept.countValue / genderCountResults[0].countValue) * 100)
+                    .toFixed(1);
+                }
+                const totCount = (genderCountResults[0].countValue <= 20) ? '&le; 20'
+                : genderCountResults[0].countValue;
+                genderCountTooltip += 'Total Count = <strong>' + totCount + '</strong>';
+          }
+          const count = (concept.countValue <= 20) ? '&le; 20' : concept.countValue;
+          pointData.push({
+            toolTipHelpText: '<div class="bio-sex-tooltip"><strong>' +
+              count + '</strong>' +
+              ' participants <br> who had <strong>' + concept.analysisStratumName +
+              '</strong> as sex assigned at birth and that is <strong>' +
+               percentage + '% </strong> of all participants with the same criteria. ('
+               + genderCountTooltip + ')</div>',
+            name: concept.analysisStratumName,
+            y: concept.countValue,
+            concept: '',
+            analysisId: ''
+          });
+          categoryArr.push(concept.analysisStratumName);
+        }
+        pointData = pointData.sort((a, b) => {
+          if (a.name > b.name) {
+            return 1;
+          }
+          if (a.name < b.name) {
+            return -1;
+          }
+          return 0;
+        }
+        );
+        categoryArr = categoryArr.sort((a, b) => {
+          if (a > b) {
+            return 1;
+          }
+          if (a < b) {
+            return -1;
+          }
+          return 0;
+        });
```

Comment:
Same as above `sort` comments:
```suggestion
        pointData.sort((a, b) => a.name.localeCompare(b.name));
        categoryArr.sort();
```

Reply:
Changed this.